src/features/breadcrumbs/atoms/breadcrumbsItemsAtom.ts

```diff
@@ -1,7 +1,40 @@
-import { action, atom } from '@reatom/core';
-import { currentMapPositionAtom } from '~core/shared_state';
+import { action, type AsyncCtx, atom, reatomAsync, withAbort } from '@reatom/framework';
+import { debounce } from '@github/mini-throttle';
+import { setCurrentMapBbox, type Bbox } from '~core/shared_state/currentMapPosition';
 import { getBboxForGeometry } from '~utils/map/camera';
-import type { Bbox } from '~core/shared_state/currentMapPosition';
+import { getBoundaries } from '~core/api/boundaries';
+import { isAbortError } from '~core/api_client/errors';
+import { getCenterFromPosition } from '../helpers/breadcrumbsHelpers';
+import type { CurrentMapPositionAtomState } from '~core/shared_state/currentMapPosition';
+
+const debouncedItemsFetch = debounce(
```

Comment:
Reatom has debounce api as well

Reply:
It will looks something like
```ts
import { action, atom, reatomAsync, withAbort } from '@reatom/framework';
import { debounce } from '@reatom/lens';
import { setCurrentMapBbox, type Bbox } from '~core/shared_state/currentMapPosition';
import { getBboxForGeometry } from '~utils/map/camera';
import { getBoundaries } from '~core/api/boundaries';
import { isAbortError } from '~core/api_client/errors';
import { getCenterFromPosition } from '../helpers/breadcrumbsHelpers';
import type { CurrentMapPositionAtomState } from '~core/shared_state/currentMapPosition';

export const breadcrumbsItemsAtom = atom<GeoJSON.Feature[] | null>(
  null,
  'breadcrumbsAtom',
);

export const fetchBreadcrumbsItems = reatomAsync(
  async (ctx, position: CurrentMapPositionAtomState) => {
    if (!position) return;

    try {
      const coords: [number, number] = getCenterFromPosition(position);
      const response = await getBoundaries(coords, ctx.controller);

      if (response) {
        breadcrumbsItemsAtom(ctx, response.features);
      } else {
        breadcrumbsItemsAtom(ctx, null);
      }
    } catch (error) {
      if (!isAbortError(error)) {
        console.error('Error when trying to retrieve boundaries:', error);
      }
    }
  },
  'breadcrumbsItemsResource',
).pipe(
  debounce(1000), // Дебаунс с задержкой 1000 мс
  withAbort()
);

export const onBreadcrumbClick = action((ctx, value: string | number) => {
  const items = ctx.get(breadcrumbsItemsAtom);
  if (!items) return;
  const item = items.find((item) => item.id === value);
  if (!item) {
    console.error(`Breadcrumb item with value ${value} not found.`);
    return;
  }

  const bbox = getBboxForGeometry(item.geometry);
  if (bbox) {
    setCurrentMapBbox(ctx, bbox);
  }
}, 'onBreadcrumbClick');

export const onZoomToWholeWorld = action((ctx) => {
  const bbox: Bbox = [-180, -80, 180, 80];
  setCurrentMapBbox(ctx, bbox);
}, 'onZoomToWholeWorld');
```